source/include/shadow.h

```diff
@@ -470,10 +548,14 @@ typedef enum ShadowStatus
  * uint16_t outLength = 0;
  * const char * pThingName = "TestThingName";
  * uint16_t thingNameLength  = ( sizeof( thingName ) - 1U );
+ * const char * pShadowName = "TestShadowName";
+ * uint16_t shadowNameLength  = ( sizeof( shadowName ) - 1U );
```

Comment:
```suggestion
 * uint16_t shadowNameLength  = ( sizeof( pShadowName ) - 1U );
```

Reply:
And by block copying the error, I carried it forward to here. Similarly fixed.